images/build/setcap/Makefile

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+.PHONY:	build push all all-build all-push-images all-push push-manifest
+
+REGISTRY?="gcr.io/k8s-staging-build-image"
+IMAGE=$(REGISTRY)/setcap
+
+TAG ?= $(shell git describe --tags --always --dirty)
+IMAGE_VERSION ?= buster-v1.4.0
+CONFIG ?= buster
+DEBIAN_BASE_VERSION ?= buster-v1.4.0
+
+ARCH?=amd64
+ALL_ARCH = amd64 arm arm64 ppc64le s390x
+
+BASE_REGISTRY?=k8s.gcr.io/build-image
+BASEIMAGE?=$(BASE_REGISTRY)/debian-base-$(ARCH):$(DEBIAN_BASE_VERSION)
+
+# Build args
+QEMUVERSION=5.2.0-2
+
+# This option is for running docker manifest command
+export DOCKER_CLI_EXPERIMENTAL := enabled
+
+build:
+ifneq ($(ARCH),amd64)
```

Comment:
Let's do not silently assume that we're running on amd64.
```suggestion
```

Reply:
Do we build these images on another arch?
NOTE: this is from multiple other makefiles in this repo:
https://github.com/kubernetes/release/blob/aab9759033fd04c2ca71a5b4b3a6734d36f678a9/images/build/debian-iptables/Makefile#L45
https://github.com/kubernetes/release/blob/aab9759033fd04c2ca71a5b4b3a6734d36f678a9/images/build/debian-hyperkube-base/Makefile#L75
https://github.com/kubernetes/release/blob/aab9759033fd04c2ca71a5b4b3a6734d36f678a9/images/build/debian-base/Makefile#L115

Reply:
> Do we build these images on another arch?

Not in the CI, but when developing those images locally different machines using different architectures would result in different outputs.

> IMO if we consider this a problem this needs a cleanup orthogonal to this PR and the discussion around what way to use setcap

Right now I'm working on cleaning up the debian-base image in https://github.com/kubernetes/release/pull/1909. Sure, I'm happy to follow-up on that later on.

Reply:
fair enough! thanks :-)